setup.py

```diff
@@ -16,39 +16,53 @@
 
 import os
 import sys
+import glob
+import platform
+import shutil
 from setuptools import find_packages
 from setuptools import setup, Extension
 from setuptools.command.install import install
 from setuptools.command.build_ext import build_ext
 
+VERSION = '2.10b3'
```

Comment:
This line is used for downloading dlls from github releases and will be removed once the wheels are automatically uploaded to PyPI.